functions/Element/getElementPosition.yaml

```diff
@@ -0,0 +1,30 @@
+shared:
+  name: 'getElementPosition'
+  pair: 'setElementPosition'
+  description: |
+    This function allows you to retrieve the position coordinates of an element. This can be any real world element, including:
+    - Players
+    - Vehicles
+    - Objects
+    - Pickups
+    - Markers
+    - Collision shapes
+    - Blips
+    - Radar areas
+    - Syntax
+  returns:
+    description: |
+      Returns three floats indicating the position of the element, x, y and z respectively.
```

Comment:
In the original source from the wiki, "float", "x", "y", and "z" are cursive. Use Markdown formatting, like `*float*s`